cocoeval.py

```diff
@@ -509,8 +507,8 @@ def setDetParams(self):
         self.iouThrs = np.linspace(.5, 0.95, np.round((0.95 - .5) / .05) + 1, endpoint=True)
         self.recThrs = np.linspace(.0, 1.00, np.round((1.00 - .0) / .01) + 1, endpoint=True)
         self.maxDets = [1, 10, 100]
-        self.areaRng = [[0 ** 2, 1e5 ** 2], [0 ** 2, 20 ** 2], [20 ** 2, 96 ** 2], [96 ** 2, 1e5 ** 2]]
-        self.areaRngLbl = ['all', 'small', 'medium', 'large']
+        self.areaRng = [[0 ** 2, 1e5 ** 2], [0 ** 2, 14 ** 2], [14 ** 2, 1e5 ** 2]]
```

Comment:
@taraspiotr this SMALL_ANNOTATIONS_SIZE should be also available here because otherwise our scoring would be off.

Reply:
@jakubczakon we change this parameters in coco_evaluation() function in utils.py. 14 is set as a default but is overridden.